docs/samples/browser-plugin-meetings/app.js

```diff
@@ -703,6 +703,8 @@ function cleanUpMedia() {
   // the local tracks get stopped when we get the unpublished event for them, so here we only need to lose the reference to them
   localMedia.microphoneTrack = undefined;
   localMedia.cameraTrack = undefined;
+  meetingStreamsLocalAudio.srcObject = null;
```

Comment:
maybe it would be better to do this inside the handler for when the track is unpublished (in `handleTrackUnpublished()`) - that's also where we stop it.

Reply:
Yeah updated the code for both `meetingStreamsLocalAudio` and `meetingStreamsLocalVideo`